docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -368,10 +372,25 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 
 > External access type to interactive jobs. Options are:
 >
->  * `portforward` (deprecated)
->  * `loadbalancer`
 >  * `nodeport`
 >  * `external-url`
+>
+> This flag can be used both in the standard format which uses command flag for all the options.
+>
+> For example:
+>
+> `runai submit test-jup -p team-a -i gcr.io/run-ai-demo/jupyter-tensorboard --service-type external-url --port 8888`
+>
+> `runai submit test-np -p team-a -i ubuntu --service-type nodeport --port 30000:7070`
+>
+> However, you can also use the newer format, where the options are listed in CSV style using multiple instances of the same option and commas to seperate the values for them.
+>
+> For example:
+>
+>`runai submit test-np -p team-a -i ubuntu --service-type nodeport,port=30000:7070 --service-type external-url,port=88880`
```

Comment:
we should either explain which parameters (e.g port) should/must be added to each service-type, or give examples that cover all the options.
specifically, what we currently miss is the custom-url parameter of the external-url type. it's optional.
I'm not referring to --custom-url flag, but to its twin for the case of using the CSV format.

Reply:
this is not correct. 'external-url' is not a parameter, it's the service name. and 'custom-url' is an OPTIONAL parameter so it's not accurate to say it "needs to be added".

Reply:
@shayasoolin I added to the second example and changed the suggested wording as per your explanation.